com.github.emmanueltouzery.projectpad.json

```diff
@@ -0,0 +1,99 @@
+{
+  "app-id": "com.github.emmanueltouzery.projectpad",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "projectpad",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--share=network",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/projectpad/cargo"
+    }
+  },
+  "modules": [
+    {
+      "name": "gtksourceview-3.24.11",
+      "only-arches": [],
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd",
+          "url": "http://ftp.gnome.org/pub/gnome/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz"
+        }
+      ]
+    },
+    {
+      "name": "tcl-8.6.10",
+      "only-arches": [],
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && ./configure --prefix=/app --enable-shared=no",
+        "cd unix && make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "5196dbf6638e3df8d5c87b5815c8c2b758496eb6f0e41446596c9a4e638d87ed",
+          "url": "https://prdownloads.sourceforge.net/tcl/tcl8.6.10-src.tar.gz"
+        }
+      ]
+    },
+    {
+      "name": "sqlcipher-3.4.2",
+      "only-arches": [],
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --enable-tempstore=yes CFLAGS=\"-DSQLITE_HAS_CODEC\" LDFLAGS=\"-lcrypto\" --prefix=/app",
+        "make install"
+      ],
```

Comment:
Same here & pass config-options.

Reply:
I tried this:

```json
      "config-opts": [
        "--enable-tempstore=yes",
        "CFLAGS=\"-DSQLITE_HAS_CODEC\"",
        "LDFLAGS=\"-lcrypto\""
]
```
but it failed, saying that the C compiler cannot create executables. I think the CFLAGS and LDFLAGS must be passed in another way, but I'm not sure how?

Reply:
@bilelmoussaoui any suggestion?

Reply:
Those should be set as build-options. Check how other apps do that

Reply:
I managed to do that in the other cases for this app, but this one eludes me. but ok, i'll try more.

Reply:
ok, I finally made it work. I couldn't make file extension filtering, but I added a check for wrong file extensions in the app. The latest manifest doesn't ask for the home permissions anymore.

Reply:
managed to make also that last case work. the problem were the quotes (even escaped), removing them fixed it.